Custom-Audio-Driver/Custom-Audio-Driver/DefaultAudioDevice.swift

```diff
@@ -655,11 +659,11 @@ func updateRecordingDelay(withAudioDevice audioDevice: DefaultAudioDevice) {
         let session = AVAudioSession.sharedInstance()
         let interval = session.inputLatency
         
-        audioDevice.recordingDelayHWAndOS += UInt32(interval * 1000000)
+        audioDevice.recordingDelayHWAndOS += UInt32(interval * DefaultAudioDevice.kToMicroSecond)
         let ioInterval = session.ioBufferDuration
         
-        audioDevice.recordingDelayHWAndOS += UInt32(ioInterval * 1000000)
-        audioDevice.recordingDelayHWAndOS += UInt32(audioDevice.recordingAudioUnitPropertyLatency * 1000000)
+        audioDevice.recordingDelayHWAndOS += UInt32(ioInterval * DefaultAudioDevice.kToMicroSecond)
+        audioDevice.recordingDelayHWAndOS += UInt32(audioDevice.recordingAudioUnitPropertyLatency * DefaultAudioDevice.kToMicroSecond)
```

Comment:
Can't we add all of them in one line like  " kToMicroSecond * (a+ b +c )"

Reply:
In my opinion it would indeed make more sense, but in all other SDK samples and even in our native repo we have it like this

Reply:
the legacy continues ....